.template/addons/github/.github/workflows/README.md.tt

```diff
@@ -17,3 +17,43 @@ Depending on the workflow used in the project, set up the following secrets:
 - DOCKER_IMAGE (Default: ${{ github. repository }})
 - HEROKU_API_KEY (for deployment)
 - DANGER_GITHUB_API_TOKEN (Default: ${{ github.token }})
+
+### Heroku Deployment
+
+Use the `deploy_heroku.yml` GitHub Action to deploy your app in Heroku.
+
+
+- If needed, create an account on [Heroku.com](https://www.heroku.com/)
+- Add your Heroku API Key (`HEROKU_API_KEY`) in GitHub Secrets
+- Create 2 new Heroku apps:
+    - {your-project-name}
+    - {your-project-name}-staging
+- If needed, update the `.github\workflow\deploy_heroku.yml` file to matches your Heroku apps names:
+```yml
+[...]
+      - name: Set env HEROKU_APP
+        run: |
+          if [[ $BRANCH_TAG = "latest" ]]
+          then
+            echo "HEROKU_APP=HERE" >> $GITHUB_ENV
+          else
+            echo "HEROKU_APP=HERE-staging" >> $GITHUB_ENV
+          fi
+[...]
+```
+
+> This step might simply be changing '_' to '-' in the app name
+
+- Now you still need to attach a database to **each** app.
+    - From [Heroku Dashboard](https://dashboard.heroku.com/apps), open the app
+    - Go to the 'Resources' tab
+    - In the 'Add-ons' section, search for 'Postgres' and select 'Heroku Postgres'
+    - You can select a free plan to start (Hobby Dev - Free) and 'Submit Order Form'
```

Comment:
Nested list should be formatted like this. More info can be found [here](https://docs.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#nested-lists).
While not really related, we also have it as a [convention](https://nimblehq.co/compass/intro/contribute/#markdown) when writing Compass.
```suggestion
- Now you still need to attach a database to **each** app.
  - From [Heroku Dashboard](https://dashboard.heroku.com/apps), open the app
  - Go to the 'Resources' tab
  - In the 'Add-ons' section, search for 'Postgres' and select 'Heroku Postgres'
  - You can select a free plan to start (Hobby Dev - Free) and 'Submit Order Form'
```